src/app/clusters/groups-server/groups-server.cpp

```diff
@@ -234,8 +234,8 @@ bool emberAfGroupsClusterGetGroupMembershipCallback(uint8_t groupCount, uint8_t
             status = emberGetBinding(i, &entry);
             if ((status == EMBER_SUCCESS) && (entry.type == EMBER_MULTICAST_BINDING) && (entry.local == emberAfCurrentEndpoint()))
             {
-                list[listLen]     = LOW_BYTE(entry.groupId);
-                list[listLen + 1] = HIGH_BYTE(entry.groupId);
+                list[listLen]     = EMBER_LOW_BYTE(entry.groupId);
+                list[listLen + 1] = EMBER_HIGH_BYTE(entry.groupId);
```

Comment:
Adding a reference to this issue, for when this is done :) 
https://github.com/project-chip/connectedhomeip/issues/1610